resources/providers/PingIdentity/Provider.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+namespace SocialiteProviders\PingIdentity;
+
+use GuzzleHttp\RequestOptions;
+use SocialiteProviders\Manager\OAuth2\AbstractProvider;
+use SocialiteProviders\Manager\OAuth2\User;
+
+class Provider extends AbstractProvider
+{
+    public const IDENTIFIER = 'PINGIDENTITY';
+
+    /**
+     * {@inheritdoc}
+     */
+    protected $scopes = [
```

Comment:
I'm still concerned about the usage of inheritdoc.  Is there a reason why these can't just be regular type annotations?  It would be better to use native PHP type annotations wherever possible, because they are enforced by the interpreter, instead of PHPDoc comments which are only suggestions.